node/typings.d.ts

```diff
@@ -156,8 +160,16 @@ interface Price {
   id: string
 }
 
+interface CustomField {
+  name: string
+  type: 'text'
+  value?: string
+}
+
 interface B2BSettingsInput {
   autoApprove: boolean
   defaultPaymentTerms: PaymentTerm[]
   defaultPriceTables: Price[]
+  defaultOrganizationCustomFields: CustomField[]
+  defaultCostCenterCustomFields: CustomField[]
```

Comment:
These should use a different type called `CustomFieldSetting` or something which doesn't have a `value` field.

Reply:
You are correct, there should be a separate interface for that case ✅